tools/websoc-api-next/index.ts

```diff
@@ -11,6 +12,7 @@ import {
   Term,
   TermData,
   WebsocAPIResponse,
+  WebsocSectionMeeting,
 } from "peterportal-api-next-types";
 
 /* region Constants */
```

Comment:
```suggestion
//-----------------------------------------------------------------------------------
// START Constants
//-----------------------------------------------------------------------------------
```
My engineering verilog classes had lots of comments like this and they've kinda grown on me lol